__tests__/db/costumes.test.js

```diff
@@ -6,15 +6,15 @@ const {
     updateCostume, 
     deleteCostumeById,
     getPool
-} = require("../index");
+} = require("../../db/index");
```

Comment:
for future reference - these changes feel like they should be in a separate PR from feature changes. It is harder to review a PR with multiple types of changes, which makes it easier for bugs to sneak in